test/fixtures/v1-vt-05401-state-utility-lowincome.json

```diff
@@ -642,26 +819,100 @@
       },
       "amount": {
         "type": "dollar_amount",
-        "number": 500,
+        "number": 800,
         "maximum": 800
       },
       "owner_status": [
         "homeowner"
       ],
-      "start_date": "2022",
-      "end_date": "2023",
+      "start_date": "2022-01-01",
+      "end_date": "2023-12-31",
       "ami_qualification": null,
       "eligible": true,
-      "short_description": "BED customers may receive up to $800 rebate for a HPWH."
+      "short_description": "BED customers may receive upto $800 rebate for a HPWH."
+    },
+    {
+      "payment_methods": [
+        "rebate"
+      ],
+      "authority_type": "state",
+      "authority": "vt-efficiency-vermont",
+      "program": "Efficiency Vermont - Residential Energy Efficiency Rebate Program",
+      "program_url": "https://www.efficiencyvermont.com/rebates/list/integrated-controls",
+      "item": {
+        "type": "other",
+        "name": "Other"
+      },
+      "amount": {
+        "type": "dollar_amount",
+        "number": 600,
+        "maximum": 600
+      },
+      "owner_status": [
+        "homeowner"
+      ],
+      "start_date": "2021-12-01",
+      "ami_qualification": null,
+      "eligible": true,
+      "short_description": "Install integrated controls with a new ductless heat pump in a single family home and get up to $600 back."
+    },
+    {
+      "payment_methods": [
+        "rebate"
+      ],
+      "authority_type": "state",
+      "authority": "vt-efficiency-vermont",
+      "program": "Efficiency Vermont Heat Pump Rebate Income Bonus",
+      "program_url": "https://www.efficiencyvermont.com/Media/Default/docs/rebates/forms/efficiency-vermont-bonus-verification.pdf",
+      "item": {
+        "type": "heat_pump_air_conditioner_heater",
+        "name": "Heat Pump Air Conditioner/Heater"
+      },
+      "amount": {
+        "type": "dollar_amount",
+        "number": 500,
+        "maximum": 500
+      },
+      "owner_status": [
+        "homeowner"
+      ],
+      "ami_qualification": null,
+      "eligible": false,
+      "short_description": "$500 bonus rebate for income-eligible Vermonters to work with an Efficiency Excellence Network contractor to install an air-to-water heat pump."
+    },
+    {
+      "payment_methods": [
+        "rebate"
+      ],
+      "authority_type": "state",
+      "authority": "vt-efficiency-vermont",
+      "program": "Efficiency Vermont Heat Pump Rebate Income Bonus",
+      "program_url": "https://www.efficiencyvermont.com/Media/Default/docs/rebates/forms/efficiency-vermont-bonus-verification.pdf",
+      "item": {
+        "type": "geothermal_heating_installation",
+        "name": "Geothermal Heating Installation"
+      },
+      "amount": {
+        "type": "dollar_amount",
+        "number": 500,
+        "maximum": 500
+      },
+      "owner_status": [
+        "homeowner"
+      ],
+      "start_date": "2021-07-01",
+      "ami_qualification": null,
+      "eligible": false,
```

Comment:
These two are ineligible because they're add-ons to the EVT air-to-water and GSHP incentives, which are both excluded by the BED ones, which are eligible.